app/templates/package.json

```diff
@@ -2,19 +2,25 @@
   "name": "<%= moduleName %>",
   "version": "<%= moduleVersion %>",
   "description": "<%= moduleDesc %>",
-  "main": "index.es5.js",
+  "main": "dist/index.js",
+  "files": [
+    "bin/",
+    "dist/",
+    "LICENSE"
+  ],
   "scripts": {
-    "coverage": "isparta cover _mocha index.js -- --require babel/register",
-    "precoveralls": "npm run coverage",
-    "coveralls": "coveralls < coverage/lcov.info",
-    "test": "mocha --require babel/register",
+    "clean": "rimraf dist",
+    "lint": "eslint src test",
+    "build": "npm run clean && npm run lint && babel --optional runtime src -d dist",
+    "test": "npm run build && babel-node test/*.js",
```

Comment:
The more I think about it, the problem with npm scripts is that while on one hand they are simple to reason about, relatively short and all affordably written in your package.json (which is really nice), they are very tightly couples, it's like a micro DSL for your own app. If you can memorize your preferred script workflow you'll be okay for a while, but it's not really flexible.

I am thinking of adding an option to the other generator to scaffold out a project with a Flyfile, maybe later when fly 1.0 lands early next month with the complete feature set and a stable API.